### PR TITLE
Created a pylintrc file

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Created a .pylintrc file using 
```js
pylint --generate-rcfile > .pylintrc
```
Now we can modify the same for the warnings that are not actually errors

The warnings that I removed by adding them in the .pylintrc :
```js
disable=E1101,
        E0401,
        R0801,
        R0903,
        C0103,
        W0703,
        R0902
```
Updates: #58
Signed-off-by: aujjwal-redhat <aujjwal@redhat.com>

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -20,12 +20,13 @@ jobs:
...
+
       - name: Lint with flake8
         run: |
           flake8 $(pwd) --count --select=E9,F63,F7,F82 --show-source --statistics
           flake8 $(pwd) --count --max-complexity=10 --max-line-length=79 --statistics
       - name: Lint with Pylint
         run: |
           pylint $(pwd)
-          rm __init__.py
+          pylint $(pwd)/redant_libs/
```
Comment (aujjwal-redhat) [inline .github/workflows/linting.yml:31]: This is temporary. We need to create packages. Then we won't need to provide absolute paths.

**Comment by nik-redhat** [comment]
Could you also fix the errors being reported by the pylint in the workflow?

**Comment by aujjwal-redhat** [comment]
> Could you also fix the errors being reported by the pylint in the checks?

Yeah sure

**Comment by aujjwal-redhat** [comment]
Errors & Messages:

E1101:
'%s %r has no %r member%s'
https://pycodequ.al/docs/pylint-messages/e1101-no-member.html

In our code
E1101: Instance of 'VolumeOps' has no 'rlog' member (no-member)

E0401:
 Unable to import 'redant_libs.support_libs.rexe' (import-error)

R0801:
R0801: Similar lines in 5 files

R0903,

C0103,
C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
W0703:
W0703: Catching too general exception Exception (broad-except)

R0902
R0902: Too many instance attributes (12/7) (too-many-instance-attributes)

These were not errors actually. 
Few among these which I felt we can still have are:
W0703: Catching too general exception Exception (broad-except)
Coz we might need to have more specific exceptions rather than
```js
except Exception as e
```
and
C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
Coz it might help in making the code more consistent throughout the project.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+        attribute-defined-outside-init,
+        missing-docstring,
+        E1101,
+        E0401,
+        R0801,
+        R0903,
+        C0103,
+        W0703,
+        R0902
+
```
Comment (aujjwal-redhat) [inline .pylintrc:145]: Here are the warnings that are disabled.

**Comment by nik-redhat** [comment]
Is the import error that was being reported earlier fixed, or disabled?

**Comment by aujjwal-redhat** [comment]
> Is the import error that was being reported earlier fixed, or disabled?

It is disabled for now.
The issue is we need to first create packages or either add pythonpath.
Adding pytohn path will not be a good choice as compared to creating packages.
That's why added the following too:
https://github.com/srijan-sivakumar/redant/pull/88#discussion_r595923628

**Comment by aujjwal-redhat** [comment]
To regenerate the specific errors:
1. Remove the specific errors or all of them from disable in .pylintrc 
2. Then run the command:
```js
pylint -j 4 --rcfile=.pylintrc redant_libs/
```

**Comment by nik-redhat** [comment]
Okay, do we have an issue open to track this? So, that we don't forget to fix it.

**Comment by aujjwal-redhat** [comment]
> Okay, do we have an issue open to track this? So, that we don't forget to fix it.

Yeah I will add one with all the required info.

**Comment by nik-redhat** [comment]
@srijan-sivakumar does it look fine to you?

**Comment by aujjwal-redhat** [comment]
Added the issue #89 
@nik-redhat

File: .github/workflows/linting.yml (hunk 1 above)
Comment (srijan-sivakumar) [reply .github/workflows/linting.yml:31]: What about the test cases ?

File: redant_libs/support_libs/rexe.py [hunk 3]
```diff
@@ -83,16 +83,14 @@ def execute_command(self, node, cmd):
         if not self.connect_flag:
             ret_dict['Flag'] = False
             return ret_dict
-        stdin, stdout, stderr = self.node_dict[node].exec_command(cmd)
```
Comment (srijan-sivakumar) [inline redant_libs/support_libs/rexe.py:86]: No. this is not the way! rather it should be `_, stdout, stderr = self.node_dict[node].exec(cmd)`

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: .github/workflows/linting.yml (hunk 1 above)
Comment (aujjwal-redhat) [reply .github/workflows/linting.yml:31]: Added now.

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**